web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1086,7 +1093,7 @@ $.extend(fixmystreet.set_up, {
     $('.js-multiple').make_multi();
 
     function update_label(id, str) {
-        $(id).prev('label').addClass('hidden-js').after(function(){ return $('<span>' + this.innerHTML + '</span>'); });
+        $(id).prev('label').addClass('hidden-js').attr('for', id.slice(1)).after(function(){ return $('<span>' + this.innerHTML + '</span>'); });
```

Comment:
This needs to be prevAll because the filter announcement gets in the way otherwise